app/controllers/server_load_readings_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ServerLoadReadingsController < ServerBaseController
+  def index
+  end
+
+  def data
```

Comment:
Would it make sense to call this method `Show`? To keep in line with the REST routes